src/models/User.js

```diff
@@ -0,0 +1,75 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const UserSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: [true, "Please Provide Your First Name"],
+    maxlength: [50,"First name should not be more that 50 characters"],
+    minlength: [3, "First name must be at least 3 characters"],
+  },
+  lastName: {
+    type: String,
+    required: [true, "Please Provide Your Last Name"],
+    maxlength: 50,
```

Comment:
maxlength: [50,"Last name should not be more that 50 characters"],

Reply:
Done